client/wildcard/src/components/Feedback/FeedbackPrompt/FeedbackPrompt.tsx

```diff
@@ -149,61 +143,95 @@ export const FeedbackPromptContent: React.FunctionComponent<ContentProps> = ({
+                        disabled={submitting}
                     />
-                    {error && (
-                        <ErrorAlert error={error} icon={false} className="mt-3" prefix="Error submitting feedback" />
+                    {submitResponse?.errorMessage && (
+                        <ErrorAlert
+                            error={submitResponse?.errorMessage}
+                            icon={false}
+                            className="mt-3"
+                            prefix="Error submitting feedback"
+                        />
                     )}
                     <Button
-                        disabled={!rating || !text || loading}
+                        disabled={!rating || !text || submitting}
                         role="menuitem"
                         type="submit"
                         display="block"
                         variant="secondary"
                         className={styles.button}
                     >
-                        {loading ? <LoadingSpinner /> : 'Send'}
+                        {submitting ? <LoadingSpinner /> : 'Send'}
                     </Button>
                 </Form>
             )}
         </>
     )
 }
 
-interface Props {
-    open?: boolean
-    routes: readonly LayoutRouteProps<{}>[]
+interface FeedbackPromptTriggerProps {
+    isOpen?: boolean
+    onClick?: React.MouseEventHandler<HTMLElement>
+}
+
+interface FeedbackPromptProps extends FeedbackPromptContentProps {
+    /**
+     * Determines if the prompt is opened by default
+     *
+     * @default false
+     */
+    openByDefault?: boolean
+    position?: Position
+    modal?: boolean
+    modalLabelId?: string
+    children: React.FunctionComponent<FeedbackPromptTriggerProps> | ReactNode
 }
 
-export const FeedbackPrompt: React.FunctionComponent<Props> = ({ open, routes }) => {
-    const [isOpen, setIsOpen] = useState(() => !!open)
-    const handleToggle = useCallback(() => setIsOpen(open => !open), [])
-    const forceClose = useCallback(() => setIsOpen(false), [])
-    const match = useRoutesMatch(routes)
+export const FeedbackPrompt: React.FunctionComponent<FeedbackPromptProps> = ({
+    openByDefault = false,
+    onSubmit,
+    children,
+    onClose,
+    position = Position.bottomEnd,
+    modal = false,
+    modalLabelId = 'sourcegraph-feedback-modal',
+}) => {
+    const [isOpen, setIsOpen] = useState(() => !!openByDefault)
+    const ChildrenComponent = typeof children === 'function' && children
+    const handleClosePrompt = useCallback(() => {
+        setIsOpen(false)
+        onClose?.()
+    }, [onClose])
+
+    const toggleIsOpen = useCallback(() => {
+        setIsOpen(isOpen => !isOpen)
+    }, [])
+
+    const triggerElement = ChildrenComponent ? <ChildrenComponent isOpen={isOpen} onClick={toggleIsOpen} /> : children
+
+    const contentElement = (
+        <FeedbackPromptContent onSubmit={onSubmit} productResearchEnabled={true} onClose={handleClosePrompt} />
```

Comment:
```suggestion
    modalLabelId = 'sourcegraph-feedback-modal',
    productResearchEnabled,
}) => {
    const [isOpen, setIsOpen] = useState(() => !!openByDefault)
    const ChildrenComponent = typeof children === 'function' && children
    const handleClosePrompt = useCallback(() => {
        setIsOpen(false)
        onClose?.()
    }, [onClose])

    const toggleIsOpen = useCallback(() => {
        setIsOpen(isOpen => !isOpen)
    }, [])

    const triggerElement = ChildrenComponent ? <ChildrenComponent isOpen={isOpen} onClick={toggleIsOpen} /> : children

    const contentElement = (
        <FeedbackPromptContent onSubmit={onSubmit} productResearchEnabled={productResearchEnabled} onClose={handleClosePrompt} />
```

We only want to show the product research message for the prompt in the nav bar